orttraining/orttraining/test/python/orttraining_ortmodule_tests.py

```diff
@@ -62,6 +62,19 @@ def run_ortmodule_poc_net(cwd, log, no_cuda, data_dir):
 
     run_subprocess(command, cwd=cwd, log=log).check_returncode()
 
+
+def run_ortmodule_torch_lightning(cwd, log, no_cuda, data_dir):
```

Comment:
I think add call to ```main``` was inadvertently missed?

Reply:
Yes and no. Newer `torchtext` breaks PyTorch lightning, but the call to it should be commented out and not deleted :)